components/bl-markdown-component/src/index.js

```diff
@@ -0,0 +1,62 @@
+import MarkdownIt from './markdown-it.min.js';
+import { useState, useEffect } from 'react';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function MyCustomComponent({ component }) {
+  const { classList, style, display, url, markdownText, height, width } = component;
+
+  const [markdown, setMarkdown] = useState('');
+  const [isLoad, setIsLoad] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  useEffect(() => {
+    const md = new MarkdownIt();
```

Comment:
I'm not sure about creating new instance of library every changing of this properties.
Could it be created only once at first render?